lib/codemod/src/transforms/__testfixtures__/convert-to-module-format/story-parameters.output.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+import React from 'react';
+import Button from './Button';
+
+export default {
+  title: 'Button',
+};
+
+export const story1 = () => <Button label="The Button" />;
+story1.title = 'with story parameters';
+
+story1.parameters = {
+    header: false,
```

Comment:
Not crucial, but can we format the code better?

Reply:
Done, using prettier. It's a little ugly, but better than what jscodeshift provides I suppose.